### ISSUE TITLE
Crashlytics upload dSYM successfully, But firebase console of Crashlytics still show dSYM required

### ISSUE THREAD

**Issue by ACFancy** [firebase/firebase-ios-sdk#5327]
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: __11.2.1___
  * Firebase SDK version: __6.14.0___
  * Firebase Component: __Crashlytics___ (Auth, Core, Database, Firestore, Messaging, Storage, etc)
  * Component version: __3.14.0___
  * Installation method: `CocoaPods ` 

### [REQUIRED] Step 2: Describe the problem
Crashlytics upload dSYM successfully, But firebase console of Crashlytics still show dSYM required
#### Steps to reproduce:
1. go to Pod Crashlytics.framework
2. ./upload-symbols
 -gsp /path/to/GoogleService-Info.plist -p ios /path/to/dSYM
3. console show: "Successfully uploaded Crashlytics symbols"
4. in firebase  Crashlytics console, still show dSYM required
What happened? How can we make the problem occur?
This could be a description, log/console output, etc.

image here to reproduce the problem
![image](https://user-images.githubusercontent.com/10044815/78904609-632d4c80-7aaf-11ea-928e-39fbd3081295.png)

![image](https://user-images.githubusercontent.com/10044815/78904560-4db82280-7aaf-11ea-900b-48bb459d095b.png)

**Comment by morganchen12** [comment]
Can you try this with the latest version of `upload-symbols` included in Firebase 6.22.0?

**Comment by kutsin** [comment]
Same issue. @morganchen12 6.22.0 not working.

Installing Firebase 6.22.0 (was 6.21.0)
Installing FirebaseAnalytics 6.4.1 (was 6.4.0)
Installing FirebaseCore 6.6.6 (was 6.6.5)
Installing FirebaseCoreDiagnostics 1.2.3 (was 1.2.2)
Installing FirebaseCrashlytics 4.0.0-beta.7 (was 4.0.0-beta.6)
Installing FirebaseInstanceID 4.3.3 (was 4.3.2)
Installing GoogleAppMeasurement 6.4.1 (was 6.4.0)
Installing GoogleDataTransport 5.1.1 (was 5.1.0)
Installing GoogleDataTransportCCTSupport 2.0.2 (was 2.0.1)

Tried with this script: /path/to/pods/directory/FirebaseCrashlytics/upload-symbols
 -gsp /path/to/GoogleService-Info.plist -p ios /path/to/dSYMs

Successfully uploaded Crashlytics symbols, but nothing happend

**Comment by sylvain-abadie** [comment]
@morganchen12 Same issue as well using 6.22.0...

**Comment by pot8os** [comment]
I'm facing the same issue. My app was still using `Fabric` and I did a correspond command however it doesn't show up even though the result was Successfully submitted symbols for xxxx.

**Comment by ACFancy** [comment]
@morganchen12 today, firebase console of Crashlytics become normal. So API of Crashlytics was recovered?

**Comment by pot8os** [comment]
My console got normal about 2 hours ago. Thanks.

**Comment by ZaVyLoN** [comment]
@ACFancy, hello. 
I still have this problem.

Xcode version: 11.4
CocoaPods (pod --version 1.9.1): 
- Crashlytics (3.14.0) 
- Firebase (6.18.0)

<img width="682" alt="Screen Shot 2020-04-12 at 21 19 50" src="https://user-images.githubusercontent.com/3078481/79076758-49854280-7d05-11ea-8d34-7b0643d0d50b.png">

**Comment by ACFancy** [comment]
@morganchen12 It seems console of Crashlytics API show delay?

**Comment by JasonSa1993** [comment]
I saw this error and after searching a lot, I came to the following solution:

1- pod deintegrate

2- pod cache clean --all

3- In the podfile install these versions of the following pod's

         `pod 'Firebase'`
         `pod 'FirebaseInstallations', '1.1.0'`
         `pod 'Fabric', '~> 1.10.2'`
         `pod 'Crashlytics', '~> 3.14.0'`
         `pod 'Firebase / Analytics'`

4- pod install

5- In your Build phases file

![image](https://user-images.githubusercontent.com/22629210/79280749-ca178080-7e76-11ea-82cc-32cac6346264.png)

`"${PODS_ROOT}/Fabric/run"`
`$(SRCROOT)/$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)`

6-  FirebaseApp.configure() in your App delegate didFinishLaunchingWithOptions

7- Run your app

8- Make your app crash

Hope it helps

**Comment by do784208** [comment]
When Fabric Crashlytics upgrade to the Firebase Crashlytics SDK, My project has same issue.

**Comment by BruceZhang2017** [comment]
I have the same problem. console show : Successfully uploaded Crashlytics symbols. but the firebase console still show required. who know it?

**Comment by davidhsu1115** [comment]
Same issue here, I wait almost 24 hours and Firebase still not showing any result.

**Comment by ashishkakkad8** [comment]
It take approx 1 day to display crashes on the firebase console. Very weird behaviour. Fabric was so fast...

**Comment by jedashford** [comment]
Same issue, I see the 24 hour wait, but still appears to be no indication my upload was successful. Follow this thread to see if someone finds a solution.

**Comment by kevinchau** [comment]
Same issue, using latest version of Fabric and Crashlytics pods. CLI will show upload successful, but no indication of such on the dashboard

**Comment by LucasAdam** [comment]
Same issue and it's been 4 days since we uploader our dsyms

**Comment by jaderYang** [comment]
Same issue here.

**Comment by ishnu47** [comment]
Any workaround please?

**Comment by jedashford** [comment]
I can confirm that after 24 hours we no longer see the 'required' next to the missing dsyms and we see crashes. Terrible user experience, but does seem to be working. I can't say if any of these crashes are recent or if we're getting real time data though.

**Comment by kevinchau** [comment]
It's been more than 24 hours for me, I've seen no change in regards to missing dSYMs

**Comment by morganchen12** [comment]
Hey all, the 24-hour delay was related to an outage and should be resolved now. If your dSYMs are still not listed as available or if you're having other migration troubles, please [file a support issue](http://firebase.google.com/support) so our staff can look at your project.

Closing this issue since it's not easily resolvable in GitHub comments; please file a support ticket instead of continuing to comment here.

**Comment by ilai1823** [comment]
Hi all, I followed all steps in the instructions from the firebase documentation. After initiating Crashlytic's bug(Crashlytics.sharedInstance().crash()), it successfully showed on Firebase Console but in the dSYMs tab and there was a warning (Upload missing dSYMs to see crashes from 1 version. Versions affected: 1.0.0 (1)). I uploaded missing dSYMs and waited 24 hours. Then the warning is gone but all the bugs too. Now I'm initiating bugs and they are not showing on Firebase Console.
